docs(grid) Documented expose aggregates for all fields (#1986)
telerik/blazor-docs: components/grid/grouping/aggregates.md

* [`GroupHeaderTemplate`]({%slug grid-templates-group-header%}) of a `GridColumn` - a header in the respective column that renders when the grid is grouped by that column. The `Value` field in the context carries the current group value.
* [`FooterTemplate`]({%slug grid-templates-column-footer%}) of a `GridColumn` - a grand total row of footers for the entire grid.

## Access The Aggregate Values

> You can access the aggregate values through the template `context`.
> 
> In all templates, you can access the aggregate values available for the column, in which the template is nested.
> 
> In both the `GroupHeaderTemplate` and the `GroupFooterTemplate`, the `context` has an `AggregateResults` property of a type Dictionary. This allows you to access the aggregates available for each column in the Grid.

Reviewer: ```suggestion
You can access the aggregate values through the template `context`:

* All templates expose the aggregate values for the current column.
* The `context` of the `GroupHeaderTemplate` and the `GroupFooterTemplate` has an `AggregateResults` property of a type `Dictionary<string, object>`. This dictionary allows you to access the aggregates for the other columns in the Grid.
```

                    }
                }
            </GroupHeaderTemplate>
            <GroupFooterTemplate>

>                 @*access the aggregates of the column in which the template is*@

Reviewer: ```suggestion
                @*access the aggregates of the ActiveProjects column*@
```

                All active projects: @context.Sum
                <br />

>                 @*access the aggregates of all columns*@

Reviewer: ```suggestion
                @*access the aggregates of the other columns*@
```

>                 Total employees and teams: @(context.AggregateResults[nameof(Employee.Team)].Count + context.AggregateResults[nameof(Employee.Name)].Count)
>                 <br />

Reviewer: ```suggestion
```

* You should define only aggregates that you will use to avoid unnecessary calculations that may be noticeable on large data sets.

> * If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will be thrown or you will not get the result from the aggregation.

Reviewer: ```suggestion
* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will occur or you will get a `null` value.
```

* You should define only aggregates that you will use to avoid unnecessary calculations that may be noticeable on large data sets.

> * If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will occur or you will get a `null` value.

Reviewer: ```suggestion
* If you try to use an aggregate that is not defined, you will get a `null` value.
* If you try to use an aggregate that is not compatible with `Field` type, a runtime error will occur.
```
